### DISCUSSION TITLE
git

### DISCUSSION THREAD

**Discussion by alex-inovaccion** [Launch-X-Latam/MisionFrontEnd#4] [Q&A]
si ya tengo descargado git pero lo tengo vinculado con otra cuenta de github que hago?

**Comment by OscarTinajero117** [comment]
Agrega tu correo de innovacción a tu cuenta principal de GitHub, eso hice yo

**Comment by akock** [comment]
la cuenta git puede manejar varios correos, incluye en tu cuenta actual el correo del curso

**Comment by AnaKwi** [comment]
en el mío cada que hago un git push me manda poner usuario y contraseña, tal vez debas de hacer lo mismo para manejarlo con cuentas diferentes.

**Comment by antovanFI** [comment]
Puedes cambiar las configuraciones que ya tengas en tu máquina con el comando `git config`

**Comment by doguedogue** [comment]
Para ver tu configuración actual:
`git config -l`

Para cambiar tu nombre de usuario
`git config --global user.name "John Doe"`

Para cambiar tu correo:
`git config --global user.email johndoe@example.com`

**Reply by maxjhael** [reply]
Gracias

**Comment by devonent** [comment]
Otra alternativa sería que manejes tus repositorios de GitHub utilizando una conexión SSH, creo que de esa manera no es necesario autenticarte con GitHub al utilizar Git (aunque si tienes que crear tus tokens de acceso).
